tests/client/api_restful/test_send_message.c

```diff
@@ -4,93 +4,82 @@
 #include <stdio.h>
 #include <unity/unity.h>
 
-#include "test_config.h"
-
-#include "client/api/restful/get_output.h"
-#include "client/api/restful/get_outputs_id.h"
+#include "client/api/restful/get_message.h"
 #include "client/api/restful/send_message.h"
+#include "core/models/payloads/tagged_data.h"
 #include "core/utils/byte_buffer.h"
+#include "core/utils/macros.h"
+#include "crypto/iota_crypto.h"
+#include "test_config.h"
 
-#include "core/address.h"
+#define TAG_TAG_LEN 14
```

Comment:
as the string in line#17, the length is 15, not 14.

Reply:
Done.